packages/fxa-auth-server/lib/subhub/updates.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. 
+*/
+
+'use strict';
+
+module.exports = function (log) {
+
+  return function start(messageQueue, push, db) {
+
+    function handleProfileUpdated(message) {
+      const uid = message && message.uid;
+
+      log.info('handleProfileUpdated', { uid, action: 'notify' });
+
+      return db.devices(uid)
+        .then(devices => push.notifyProfileUpdated(uid, devices))
```

Comment:
Will we also want to notify attached services that the profile data changed? push.notifyProfileUpdated will only send out push notifications to attached devices. You can see [how it's done for subscriptions](https://github.com/mozilla/fxa/blob/6be0bd05351ed4641bf8d061d906a0e16560b399/packages/fxa-auth-server/lib/routes/subscriptions.js#L131), though this is slightly different because this isn't tied to a [request and the metricsContext will be unavailable](https://github.com/mozilla/fxa/blob/6be0bd05351ed4641bf8d061d906a0e16560b399/packages/fxa-auth-server/lib/log.js#L130).

@philbooth - any ideas?

Reply:
> @philbooth - any ideas?

You mean about the `metricsContext` thing? Sending the notification without `metricsContext` seems fine to me, and semantically correct.

Reply:
No, this is copypasta from the profile update SQS client. Just haven't ripped out all the old bits yet. 😄

Reply:
> You mean about the metricsContext thing? Sending the notification without metricsContext seems fine to me, and semantically correct.

I mean the lack of request object. The first thing [`notifyAttachedServices` does is call request.gatherMetricsContext](https://github.com/mozilla/fxa/blob/6be0bd05351ed4641bf8d061d906a0e16560b399/packages/fxa-auth-server/lib/log.js#L130). We could stub it in I suppose.